practices/managing-static-assets.md

```diff
@@ -0,0 +1,20 @@
+# Managing static assets
+
+By **static assets** we mean webfonts, JavaScript libraries (like jQuery or Mathjax), and any other files that get loaded by the browser from a domain that is not ours. These static assets are characterised by:
+
+* Being essential or important for the working of a page.
+* Being updated infrequently.
+* Require some manual intervention in order to be updated.
+
+Usually these assets are added to the page as part of the application, and not through tag managers.
+
+For this kind of static resources is our choice to **self-host them in our servers** instead of including them on our pages from an external CDN.
```

Comment:
```suggestion
For this kind of static resources is our choice to **self-host them on our servers** instead of including them on our pages from an external CDN.
```